CONTRIBUTOR-ROLE.md

```diff
@@ -0,0 +1,96 @@
+# Contributor Roles
+
+## Security Team
+
+Security Team members are listed in the Metal3 user-guide under the
+[Project Security Policy](https://book.metal3.io/security_policy#metal3-security-committee-members).
+The security team is responsible for responding to all security issues reported
+to Metal3 project. The team can be reached at `metal3-security@googlegroups.com`.
+(Please do not report any security vulnerability to the committee members
+directly.) The security team ensures that the issues are mitigated and
+a responsible disclosure process is followed in a reasonable time. The security
+team has a designated Security Lead. Currently,
+[Tuomo Tanskanen](https://github.com/tuminoid) is acting as the Metal3 Security
+Lead. Below are the responsibilities of the Security team.
+
+* Responsibilities:
```

Comment:
This is still unclear if talking about the team or the lead, especially when some of the responsibilities are team's and some is lead's.